services/memory_storage/memory_storage.cpp

```diff
@@ -124,7 +124,7 @@ namespace services {
     bool memory_storage_t::check_database_(components::session::session_id_t& session, const database_name_t& name) {
         if (!is_exists_database_(name)) {
             actor_zeta::send(current_message()->sender(), this->address(), handler_id(route::execute_plan_finish), session,
-                             make_error(error_code_t::database_not_exists, "database not exists"));
+                            make_error(actor_zeta::detail::pmr::get_default_resource(), error_code_t::database_not_exists, "database not exists"));
```

Comment:
for now please use
https://en.cppreference.com/w/cpp/memory/get_default_resource

Reply:
Even though, I thinl it'll better to create aliase fot this, if we want in a future chanege it to something else it will be much easier to do in one place.